invenio_accounts/ext.py

```diff
@@ -198,6 +198,9 @@ def init_app(self, app, sessionstore=None, register_blueprint=True):
                 app.extensions['security'].confirm_register_form, app
             )
 
+        app.extensions['security'].login_form = login_form_factory(
```

Comment:
True, but I just followed the same pattern https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/ext.py#L193-L199